gulpfile.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
```

Comment:
This looks good, but I think you need to update your paths to work for this project, see my comments below.